quantum/keymap_extras/keymap_polish.h

```diff
@@ -0,0 +1,152 @@
+/* Copyright 2020
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "keymap.h"
+
+// clang-format off
+
+/*
+ * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───────┐
+ * │ ` │ 1 │ 2 │ 3 │ 4 │ 5 │ 6 │ 7 │ 8 │ 9 │ 0 │ - │ = │       │
+ * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┤
+ * │     │ Q │ W │ E │ R │ T │ Y │ U │ I │ O │ P │ [ │ ] │     │
+ * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐    │
+ * │      │ A │ S │ D │ F │ G │ H │ J │ K │ L │ ; │ ' │ \ │    │
+ * ├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴────┤
+ * │        │ Z │ X │ C │ V │ B │ N │ M │ , │ . │ / │          │
+ * ├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬────┤
+ * │    │    │    │                        │    │    │    │    │
+ * └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘
+ */
+// Row 1
+#define PL_GRV  KC_GRV  // `
+#define PL_1    KC_1    // 1
+#define PL_2    KC_2    // 2
+#define PL_3    KC_3    // 3
+#define PL_4    KC_4    // 4
+#define PL_5    KC_5    // 5
+#define PL_6    KC_6    // 6
+#define PL_7    KC_7    // 7
+#define PL_8    KC_8    // 8
+#define PL_9    KC_9    // 9
+#define PL_0    KC_0    // 0
+#define PL_MINS KC_MINS // -
+#define PL_EQL  KC_EQL  // =
+// Row 2
+#define PL_Q    KC_Q    // Q
+#define PL_W    KC_W    // W
+#define PL_E    KC_E    // E
+#define PL_R    KC_R    // R
+#define PL_T    KC_T    // T
+#define PL_Y    KC_Y    // Y
+#define PL_U    KC_U    // U
+#define PL_I    KC_I    // I
+#define PL_O    KC_O    // O
+#define PL_P    KC_P    // P
+#define PL_LBRC KC_LBRC // [
+#define PL_RBRC KC_RBRC // ]
+// Row 3
+#define PL_A    KC_A    // A
+#define PL_S    KC_S    // S
+#define PL_D    KC_D    // D
+#define PL_F    KC_F    // F
+#define PL_G    KC_G    // G
+#define PL_H    KC_H    // H
+#define PL_J    KC_J    // J
+#define PL_K    KC_K    // K
+#define PL_L    KC_L    // L
+#define PL_SCLN KC_SCLN // ;
+#define PL_QUOT KC_QUOT // '
+#define PL_BSLS KC_NUBS // (backslash)
```

Comment:
The Polish layout is based on US ASCII. I tested, both `KC_BSLS` and `KC_NUBS` work, but its more true to reality if `KC_BSLS` is used.
```suggestion
```
Moving to row 2.